config/locales/public.nl.yml

```diff
@@ -15,14 +15,18 @@ nl:
     privacy: Privacy
   form:
     bank: Bank
+    digits_field: Dit veld accepteert alleen cijfers
     finish_signup: Inschrijving afronden
     general: Algemene gegevens
     introduction: Introductie %{user}
+    invalid_email: Emailadres is niet geldig
```

Comment:
E-mail adres toch?

Reply:
Oof daar gaat mijn middelbare kennis

Reply:
Dit is exact de link die ik zelf heb gebruikt om op te zoeken hoe het alweer moest, je bent niet de enige ;p